design/snapshots.md

```diff
@@ -45,7 +45,7 @@ interface.
 ## Architecture
 
 The _Snapshot Manager_ provides an API for allocating, snapshotting and mounting
-abstract, layer-based filesytems. The model works by building up sets of
```

Comment:
The plural is actually more correct here: "*An API for mounting filesystems.*" vs. "*An API for mounting filesystem.*"